tests/tt_eager/ops/test_reduce_op.cpp

```diff
@@ -1,114 +0,0 @@
-// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
-//
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
Are we just deleting this test, or was it moved elsewhere?

Reply:
Just delete, along with the reduce op from tt_lib

Reply:
Shouldn't we re-add the same test for the ttnn version to keep existing coverage? Or is this a redundant test?

Reply:
We have tests in ttnn, these tests were not great, they would just run things w/o checking if results are valid in any way. Check the comment on line 30